capstone/scripts/extract_cites.py

```diff
@@ -45,29 +45,31 @@ def extract_citations(case, html, xml):
     clean_html_pq = parse_html(clean_text(html))
     clean_html_pq('a').remove()
 
-    # Extract cites from each paragraph:
-    for el in clean_html_pq('p[id], blockquote[id]').items():
-        el_text = el.text()
-        extracted_cites = []
-        for eyecite_cite in extract_citations_from_text(el_text, require_classes=None):
-            eyecite_cites.append(eyecite_cite)
-            extracted_cites.append(eyecite_cite)
-
-            if not isinstance(eyecite_cite, FullCitation):
-                continue
-
-            # get normalized forms of cite
-            normalized_forms = {'cite': eyecite_cite.matched_text()}
-            normalized_forms['normalized_cite'] = normalize_cite(normalized_forms['cite'])
-            normalized_forms['rdb_cite'] = eyecite_cite.corrected_citation()
-            normalized_forms['rdb_normalized_cite'] = normalize_cite(normalized_forms['rdb_cite'])
-            eyecite_cite.normalized_forms = normalized_forms
-
-            for k, v in normalized_forms.items():
-                cites_by_type[k].append(v)
-
-        if extracted_cites:
-            extracted_els.append((el.attr("id"), el_text, extracted_cites))
+    # Extract cites from each paragraph for each opinion:
+    for i, section in enumerate(clean_html_pq('.opinion').items()):
```

Comment:
assume that we don't care about citations that aren't opinions. will correct this otherwise.

Reply:
Unfortunately I think we do -- if a case has a lot of head matter you'd like to be able to click from citations in the head matter to read the cases. I'm not sure where you store those in the index though.

Reply:
dangit!

I may be wrong here, but what if we keep the annotations that seem to be getting written at https://github.com/harvard-lil/capstone/blob/develop/capstone/scripts/extract_cites.py#L126 but refuse to add them to the metadata? It seems significant to be able to distinguish when the head matter is making the citation v. when the judge is making the citation. assuming that the above code captures all the usefulness of our processing of citations in head matter, it might make sense to annotate but not count it as an extracted citation.

Otherwise, maybe we can place the citation where it was previously and rename the key `other_citations` / `head_citations` or something else. I'm not sure if we should still support queries against those citations or count them as part of the weight. In my mind they certainly seem less weightier, but this would exclude someone doing metaanalysis of head matter, i suppose.